dts/arm/st/f1/stm32f103Xg.dtsi

```diff
@@ -0,0 +1,162 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2020 Google, LLC
+ *
+ * SoC device tree include for STM32F103xG SoCs
+ * where 'x' is replaced for specific SoCs like {R,V,Z}
+ */
+
+#include <mem.h>
+#include <st/f1/stm32f103Xb.dtsi>
```

Comment:
You might want to include <st/f1/stm32f103Xe.dtsi> which is closer to the Xg.

Reply:
Good catch.